synapse/server_notices/consent_server_notices.py

```diff
@@ -121,7 +119,7 @@ def copy_with_str_subst(x, substitutions):
     if isinstance(x, dict):
         return {k: copy_with_str_subst(v, substitutions) for (k, v) in x.items()}
     if isinstance(x, (list, tuple)):
-        return [copy_with_str_subst(y) for y in x]
+        return [copy_with_str_subst(y, substitutions) for y in x]
```

Comment:
Nice catch!

Reply:
Ah, this is what I meant to call out in a comment. I'm fairly certain this is the correct change!

Reply:
Yes, seems like it.